### ISSUE TITLE
The specified procedure could not be found.

### ISSUE THREAD

**Issue by LeeSingFanda** [dachev/node-cld#27]
I have a simple script with two lines of code. 

```
var cld = require('cld');
console.log('cld loaded');
```

Running the script from command line generates error like below. Any advice, please? 

```
E:\Test>node app.js
module.js:423
return process.dlopen(module, path._makeLong(filename));
         ^

Error: The specified procedure could not be found.
\\?\E:\Test\node_modules\cld\build\Release\cld.node
at Error (native)
at Object.Module._extensions..node (module.js:423:18)
at Module.load (module.js:344:32)
at Function.Module._load (module.js:301:12)
at Module.require (module.js:354:17)
at require (internal/module.js:12:17)
at Object.<anonymous> (E:\Test\node_modules\cld\index.js:2:12)
at Module._compile (module.js:398:26)
at Object.Module._extensions..js (module.js:405:10)
at Module.load (module.js:344:32)    
```

I can confirm that the file "E:\Test\node_modules\cld\build\Release\cld.node" is existing.

**Comment by LeeSingFanda** [comment]
I have figured out why, it works with older version of Node.JS 0.10.28, but not the latest version. Kindly fix this, please.

**Comment by dachev** [comment]
@LeeSingFanda The windows build hasn't received mush love from me lately but I am intending to fix that in the upcoming months. Sorry for the delay and thank you for the patience.

**Comment by dachev** [comment]
This should be fixed in 2.6.0
